IPlayer/Player.cs

```diff
@@ -18,27 +22,49 @@ public void DisplayStatus()
 
     private void DisplayHp()
     {
-        if (this._didHpDecrease)
-        {
-            Console.ForegroundColor = ConsoleColor.Red;
-        }
-        else
+        switch (this.hpState)
         {
-            Console.ForegroundColor = ConsoleColor.Green;
+            case StateOfChange.Decreased:
+                Console.ForegroundColor = ConsoleColor.Red;
+                break;
+            case StateOfChange.Increased:
+                Console.ForegroundColor = ConsoleColor.Green;
+                break;
+            case StateOfChange.Same:
+                Console.ForegroundColor = ConsoleColor.White;
+                break;
+            default:
+                break;
         }
+        this.hpState = StateOfChange.Same;
         Narrator.Write($"{this.Hp}");
         Console.ForegroundColor = ConsoleColor.White;
     }
 
     private void DisplayLevel()
     {
-        // TODO: Implement color changing ability
+        switch (this.levelState)
+        {
+            case StateOfChange.Decreased:
+                Console.ForegroundColor = ConsoleColor.Red;
+                break;
+            case StateOfChange.Increased:
+                Console.ForegroundColor = ConsoleColor.Green;
+                break;
+            case StateOfChange.Same:
+                Console.ForegroundColor = ConsoleColor.White;
+                break;
+            default:
+                break;
+        }
```

Comment:
This code is identical to the one in DisplayHP, you should consider making a method that is parameterized on the state variable.